NVlabs/instant-ngp
# Add glow options to NeRF camera keyframes

These edits allow for rendering out the amazing "glow mode" effects, controlled with camera path keyframes.

![glow](https://user-images.githubusercontent.com/22482773/182970401-0a3cc851-f0d5-442f-8b79-8aec3f71bee8.png)

- Let me know how the interpolation behaves, happy to edit.
- I'm regularly writing JavaScript, feel free to call out any type errors.
- I'm focused on NeRFs with this PR, if this negatively impacts the other primitives in some way let me know.

To test:
1) Pull this code
2) Delete or rename your `build` folder
3) Re-run the build commands from the repo root:
`cmake . -B build`
`cmake --build build --config RelWithDebInfo -j`
4) Load a NeRF scene as normal
5) Add a camera keyframe and adjust the sliders (the code should also work with previous camera paths)

Thanks!
